src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -101,4 +104,73 @@ public void selezioneVediDettaglioButton() {
+            By chipAnnullataInTimeline = By.id("Annullata-status");
+
+            getWebDriverWait(10).withMessage("il titolo Dettaglio notifica non è visibile").until(ExpectedConditions.visibilityOfElementLocated(titleDettaglioNotificaField));
+            getWebDriverWait(10).withMessage("il bottone indietro non è visibile").until(ExpectedConditions.visibilityOfElementLocated(indietroButtonBy));
+            getWebDriverWait(10).withMessage("Dettaglio notifica non è visibile").until(ExpectedConditions.visibilityOfElementLocated(informazioniBy));
+            getWebDriverWait(10).withMessage("La sezione Documenti allegati non è visibile").until(ExpectedConditions.visibilityOfElementLocated(allegatiSection));
+            getWebDriverWait(10).withMessage("Lo stato della notificanon non è visibile").until(ExpectedConditions.visibilityOfElementLocated(statoNotificaBy));
+            if (!isSelfcare) {
+                getWebDriverWait(10).withMessage("Il sezione recapiti non è visibile").until(ExpectedConditions.visibilityOfElementLocated(sezioneRecapiti));
+            }
+            getWebDriverWait(10).withMessage("Il sezione altri documenti non è visibile").until(ExpectedConditions.visibilityOfElementLocated(altriDocumenti));
+            getWebDriverWait(10).withMessage("Il pulsante sezione attestazione opponibile non è visibile").until(ExpectedConditions.visibilityOfElementLocated(attestazione));
+            getWebDriverWait(10).withMessage("Il copy di notifica annullata non è visibile").until(ExpectedConditions.visibilityOfElementLocated(copyNotificaAnnullata));
+            getWebDriverWait(10).withMessage("La chip di notifica annullata non è visibile").until(ExpectedConditions.visibilityOfElementLocated(chipAnnullataInTimeline));
+            logger.info("Dettaglio Notifica Annullata Section caricata");
+        } catch (TimeoutException e) {
+            logger.error("Dettaglio Notifica Annullata Section non caricata con errore: " + e.getMessage());
+            Assert.fail("Dettaglio Notifica Annullata Section non caricata con errore: " + e.getMessage());
+        }
+    }
+
+    public void checkStatoTimeline(String statoTimeline){
+        try {
+            By stato = By.xpath(statoTimeline);
+            getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(stato));
+            logger.info("stato timeline checkato con successo avvenuta");
+        } catch (TimeoutException e) {
+            logger.error("checkato stato timeline non avvenuta con errore: " + e.getMessage());
+            Assert.fail("checkato stato timeline non avvenuta con errore: " + e.getMessage());
+        }
+    }
+
+    public void selezioneAvvisoPagoPa() {
+        try {
+            By checkboxAvvisoPagoPa = By.xpath("//span[@data-testid='radio-button']");
+            getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(checkboxAvvisoPagoPa));
```

Comment:
Check also if clickable by adding the `ExpectedConditions.and(conditionOne, conditionTwo)`

Reply:
done